ludwig/features/number_feature.py

```diff
@@ -288,7 +288,9 @@ def normalize(series: pd.Series) -> pd.Series:
             return pd.Series(transformed_values, index=series.index)
 
         input_series = input_df[feature_config[COLUMN]].astype(np.float32)
-        proc_df[feature_config[PROC_COLUMN]] = backend.df_engine.map_partitions(input_series, normalize)
+        proc_df[feature_config[PROC_COLUMN]] = backend.df_engine.map_partitions(
+            input_series, normalize, meta=input_series
```

Comment:
Does add the `input_series` as meta help address the issue, or was it something tried during debugging that was leftover?

Reply:
This helped address the issue– before adding this, `tests/integration_tests/test_class_imbalance_feature.py::test_imbalance_ray` was failing because proc_df was passed around as an "object" column instead of a float32 column. IIRC, later calls then presumed the array was a `Double` instead of a `Float` type.